lisa/tools/tar.py

```diff
@@ -48,6 +49,21 @@ def extract(
         if strip_components:
             # optionally strip N top level components from a tar file
             tar_cmd += f" --strip-components={strip_components}"
+
+        if skip_old_files:
```

Comment:
How about remove existing extracted files always, or always overwrites? The skip-old-files is hard to use with other tools right.

Reply:
No, we specifically don't want to overwrite the old files here. 

We _have_ to assume that changes have been applied, we will wipe out patches that have been applied to source files in the output directory otherwise. We can't just delete the output directory and start over every time, since this will potentially wipe out a build directory.

Reply:
ok, it makes sense. But the name `skip_old_files` is not accurate. Please use `skip_exist_files` for the method argument.

Reply:
I feel strongly that options should have the same name as the flag, as long as we're just doing a passthrough like this. It allows people to look up the man page for the flag without having to check the LISA tool's source to see what the option resolves to.

Reply:
It depends on the design principle. If the goal is to be same as original commands, you are right. But LISA abstracts the commands for better understanding. In this case, you explained the term "old" in the code, but I read the original document to understand it finally. If it's called "exist", it would be clearer. When the tool is implemented by BSD, Windows and other OS, it also makes sense.